packages/round-manager/src/features/common/FormWizard.tsx

```diff
@@ -19,9 +19,11 @@ export function FormWizard({ initialCurrentStep = 1, initialData = {}, steps }:
   return (
     <FormContext.Provider
       value={{
-        currentStep, setCurrentStep,
+        currentStep,
```

Comment:
question: what is the best way to test the Provider setup here, if we should test that?

Reply:
decide not to test because the Provider is very implementation-specific